cardano-crypto-class/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog for `cardano-crypto-class`
 
+## 2.2.2.0
+
+* Add `SHA512` and `SHA3_512` algorithms.
+
 ## 2.2.1.1
 
 *
```

Comment:
Sorry, my bad. It should have replaced this section since `2.2.1.1` was never released
```suggestion
```